include/lsst/afw/geom/Transform.h

```diff
@@ -189,6 +189,28 @@ class Transform {
      */
     Eigen::MatrixXd getJacobian(FromPoint const &x) const;
 
+    /**
+     * Concatenate two Transforms.
+     *
+     * @tparam LastEndpoint the starting Endpoint of the Transform that feeds
+     *                      into this one
+     * @param before the Transform to apply before this one
+     * @returns a Transform that first applies `before` to its input, and then
+     *          this Transform to the result. Its inverse shall first apply the
+     *          inverse of this Transform, then the inverse of `before`.
+     *
+     * @throws InvalidParameterErrror Thrown if `this->getFromEndpoint()` does
+     *                                not match `before.getToEndpoint()`.
+     * @exceptsafe Provides basic exception safety.
+     *
+     * More than two Transforms can be combined in series. For example:
+     *
+     *     auto pixelsToSky = pupilToSky.of(fpToPupil)
+     *                                  .of(pixelsToFp);
```

Comment:
I'm not sure it is worth changing, but I usually name transforms using `from` instead of `to` because `a_values = aFromB(b_values)`. When worded that way this example will read much more naturally.